setup.py

```diff
@@ -21,7 +21,7 @@
     version=version_for_setup_py,
     zip_safe=False,
     packages=find_packages(exclude=["test"]),
-    install_requires=["aiohttp"],
+    install_requires=["aiohttp", "dataclasses;python_version<'3.7'"],
```

Comment:
fedora 31 is on 3.7 already [ref](https://docs.fedoraproject.org/en-US/fedora/f31/release-notes/developers/Development_Python/#_python_3_8_is_now_available), so the `karapace.spec` doesn't need a new dependency

Reply:
Considering we're going to drop 3.6 (https://github.com/aiven/karapace/pull/134), do we still need to add this?

Reply:
Nope :) I will update this PR